### DISCUSSION TITLE
Сброс Totum к исходному состоянию

### DISCUSSION THREAD

**Discussion by Stillilya** [totumonline/totum-ru-issues-and-discussions#326] [Как это сделать?] [answered]
Я новичок, учусь. Делаю всё по мастерклассам - получается. Обучение классное. Но после хочется начать всё заного и удалить всё, что я сделал и начать с нуля. Как можно всё сбросить к исходному состоянию?

**Comment by erpic-ru** [comment, marked as answer]
Поудаляйте таблицы через Системные - Список таблиц, да и все. 
Ещё вроде через [консольную утилиту](https://ru.docs.totum.online/bin-totum) можно залить чистую схему или через vacuum (ни разу не пользовался, правда)

**Reply by ttm-alexeypolunin** [reply]
Общая идея правильная — удалить несистемные таблицы.

В Список таблиц есть поле категория — по нему можно фильтрануть. Но тут может быть косяк тк иногда Системные таблицы могут быть отмечены как TOTUM (например таблица настроек LDAP в каких-то версиях)

Второе: удалить Кроны из таблицы Кроны

Третье: удалить файлы с сервера, если были поля типа файл по пути `/home/totum/totum-mit/http/fls/DOMAIN` и `/home/totum/totum-mit/secureFiles/DOMAIN` если использовалась галочка Защищенный доступ к файлам в PRO.

Четвертое: выполнить обновление `bin/totum git-update` — оно восстановит изменения в системных полях (которые лучше не делать вообще).
